packages/core/src/tracing/instrumentation/protocols/_http.js

```diff
@@ -5,32 +5,51 @@
 
```

Comment:
suggestion: can we rename this file to something like `httpUtils.js` or similar?

Reply:
Oh yes, absolutely. I agree the name is nonsense, a derelict from 2016. Since the module is only about capturing http headers, I'll rename it to `captureHttpHeadersUtil.js`.